### PR TITLE
Add hardtunes importer

### PR DESCRIPTION
**Pull request by basxto**
Add importer for hardtunes.com

### CODE REVIEW

File: hardtunes_importer.user.js [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+
+// Insert button into page under label information
+function insertLink(release, release_url) {
+    let edit_note = MBImport.makeEditNote(release_url, 'HardTunes.com');
+    let parameters = MBImport.buildFormParameters(release, edit_note);
+
+    document.getElementsByClassName('social-media')[0].innerHTML += MBImport.buildFormHTML(parameters) + MBImport.buildSearchButton(release);
+}
+
+// Fixed mappings for some popular labels
```
Comment (murdos) [inline hardtunes_importer.user.js:116]: I really doubt that having an hardcoded mapping is a good idea (both for labels and artists): it's error prone and difficult to maintain.

Comment (basxto) [reply hardtunes_importer.user.js:116]: This online shop has specialized on a few genres and name clashes within closely related genres are quite unlikely.
But I have no plans for extending that list any further, it’s only meant for more popular artists, which might be on compilations more.

**Comment by murdos** [comment]
Discarding this PR, since - as previously discussed - maintaining it would be difficult since all MBIDs are hardcoded in script.

**Comment by JuniorJPDJ** [comment]
@basxto you could add hardtunes artists urls to MBz and then search by URL

**Pull request closed without merging**